docs/user-manual/export-data.md

```diff
@@ -28,6 +28,6 @@ After logging in:
 
 ![Screenshot 5](./assets/screenshot-5.png)
 
-The configurations file and/or the items file. With or without comments and history
+The configuration files and/or the items files. With or without comments and history
```

Comment:
Juste une petite question: pourquoi configuration perd son "s" mais items le garde ? 🤔

Reply:
Parce que ce sont DES fichiers de UNE configuration et ce sont DES fichiers DE PLUSIEURS items :)

Reply:
Got you ! merci !  👍